src/components/AddressLine.tsx

```diff
@@ -0,0 +1,58 @@
+import {
+  CiBookmarkCheck,
+  CiDeliveryTruck,
+  CiEdit,
+  CiMoneyCheck1,
+  CiTrash,
+} from 'react-icons/ci';
+import { AddressForProfile } from '../utils';
+
+type AddressLineProps = {
+  address: AddressForProfile;
+  showEdit: boolean;
+  onEdit: (key: string) => void;
+};
+function AddressLine(props: AddressLineProps) {
```

Comment:
for clen up code here will be better to use destructure 

example: ({ address, showEdit, onEdit }: AddressLineProps)